internal/compiler-bridge/src/main/scala/xsbt/ExtractAPI.scala

```diff
@@ -598,6 +598,8 @@ class ExtractAPI[GlobalType <: Global](
           "sbt-api: Unexpected nullary method type " + in + " in " + in.owner
         )
         Constants.emptyType
+      case MethodType(_, _) =>
+        Constants.emptyType
```

Comment:
Hit the merge button on https://github.com/sbt/zinc/pull/689?

Reply:
Went with the lesser change of making it echo instead of warning.